### ISSUE TITLE
Could not find the bundle file '/libs/flag-icon-css/css/flag-icon.min.css'

### ISSUE THREAD

**Issue by phil-enfom** [abpframework/abp#11856]
* ABP Framework version 5.2.0-rc.1
* MVC
* EF Core
* Exception message:
AbpException: Could not find the bundle file '/libs/flag-icon-css/css/flag-icon.min.css' for the bundle 'Lepton.Global'!
Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers.AbpTagHelperResourceService.ProcessAsync(ViewContext viewContext, TagHelper tagHelper, TagHelperContext context, TagHelperOutput output, List<BundleTagHelperItem> bundleItems, string bundleName)
Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers.AbpBundleTagHelperService<TTagHelper, TService>.ProcessAsync(TagHelperContext context, TagHelperOutput output)
Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>g__Awaited|0_0(Task task, TagHelperExecutionContext executionContext, int i, int count)
YTjkAc96I7byyY13o4E.TGdOod9av8tqc5TENeE+<>c__DisplayClass28_0+wEmCv3lybVmY19VkHFe.MoveNext()
Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.SetOutputContentAsync()
YTjkAc96I7byyY13o4E.TGdOod9av8tqc5TENeE.ExecuteAsync()
Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, bool invokeViewStarts)
Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderLayoutAsync(ViewContext context, ViewBufferTextWriter bodyWriter)
Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
* Steps needed to **reproduce** the problem.
Created new solution using: abp new ePlus -t app-pro --preview

Issue is that the actual path to the css is /libs/flag-icon-css/css/flag-icon**s**.min.css

**Comment by realLiangshiwei** [comment]
See: https://support.abp.io/QA/Questions/2700/New-Project-520-rc1---ABP-Suite---Missing-Components

**Comment by maliming** [comment]
What's the version of your CLI? Please try to upgrade to the RC.

**Comment by phil-enfom** [comment]
CLI Version is 5.2.0-rc.1

**Comment by phil-enfom** [comment]
The referenced file is in the folder except it has the s on the the end of 'icon'
If you rename the file it works

**Comment by maliming** [comment]
Solution:

Rename `src/Web/wwwroot/libs/flag-icon-css/css/flag-icons.min.css` to `src/Web/wwwroot/libs/flag-icon-css/css/flag-icon.min.css`

Add an empty `chart.css` file in `src/Web/wwwroot/libs/chart.js/`

**Comment by johan-developer** [comment]
> Solution:
> 
> Rename `src/Web/wwwroot/libs/flag-icon-css/css/flag-icons.min.css` to `src/Web/wwwroot/libs/flag-icon-css/css/flag-icon.min.css`
> 
> Add an empty `chart.css` file in `src/Web/wwwroot/libs/chart.js/`

This found in Blazor Wasm but chart.js folder not exists.

**Comment by sturlath** [comment]
I'm getting the reverse now with 5.2.rc2 (updating from 5.1.4)

> Volo.Abp.AbpException: Could not find the bundle file '/libs/flag-icon-css/css/flag-icons.min.css' for the bundle 'Lepton.Global'! 

If I remove the '**s**' from flag-icons.min.css this error goes away

**Comment by realLiangshiwei** [comment]
@sturlath Please try `dotnet clean`

**Comment by sturlath** [comment]
@realLiangshiwei no that does not solve it. I deleted obj/bin and after updating (through Suite "all packages) it will not work without 's'...

I see that I´m also getting this error below . There isn´t any github.**min**.css created. (I deleted the lib folders and re-ran `abp install-libs` that created the folder)

```
An exception was thrown attempting to execute the error handler.
Volo.Abp.AbpException: Could not find the bundle file '/libs/highlight.js/styles/github.min.css' for the bundle '6A6C0CDFAFE77D4AC1424C4D92347409'!
   at Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers.AbpTagHelperResourceService.ProcessAsync(ViewContext viewContext, TagHelper tagHelper, TagHelperContext context, TagHelperOutput output, List`1 bundleItems, String bundleName)
   at Volo.Abp.AspNetCore.Mvc.UI.Bundling.TagHelpers.AbpBundleTagHelperService`2.ProcessAsync(TagHelperContext context, TagHelperOutput output)
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperRunner.<RunAsync>g__Awaited|0_0(Task task, TagHelperExecutionContext executionContext, Int32 i, Int32 count)
   at AspNetCoreGeneratedDocument.Volo_Abp_AspNetCore_Mvc_UI_Widgets_Components_WidgetStyles_Default.ExecuteAsync()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewComponents.ViewViewComponentResult.ExecuteAsync(ViewComponentContext context)
   at Microsoft.AspNetCore.Mvc.ViewComponents.DefaultViewComponentInvoker.InvokeAsync(ViewComponentContext context)
   at Microsoft.AspNetCore.Mvc.ViewComponents.DefaultViewComponentHelper.InvokeCoreAsync(ViewComponentDescriptor descriptor, Object arguments)
   at Volo.Abp.AspNetCore.Mvc.UI.Widgets.AbpViewComponentHelper.InvokeAsync(Type componentType, Object arguments)
   at sAPx5n0goprLsvRuLoQ.HAEetI0udFUSNyE2hrV.<>c__DisplayClass28_0.feyERlRZ7NYqZcWrsX5.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Razor.Runtime.TagHelpers.TagHelperExecutionContext.SetOutputContentAsync()
   at sAPx5n0goprLsvRuLoQ.HAEetI0udFUSNyE2hrV.ExecuteAsync()
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageCoreAsync(IRazorPage page, ViewContext context)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderPageAsync(IRazorPage page, ViewContext context, Boolean invokeViewStarts)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderLayoutAsync(ViewContext context, ViewBufferTextWriter bodyWriter)
   at Microsoft.AspNetCore.Mvc.Razor.RazorView.RenderAsync(ViewContext context)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ViewContext viewContext, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewExecutor.ExecuteAsync(ActionContext actionContext, IView view, ViewDataDictionary viewData, ITempDataDictionary tempData, String contentType, Nullable`1 statusCode)
   at Microsoft.AspNetCore.Mvc.ViewFeatures.ViewResultExecutor.ExecuteAsync(ActionContext context, ViewResult result)
   at Microsoft.AspNetCore.Mvc.ViewResult.ExecuteResultAsync(ActionContext context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResultFilterAsync>g__Awaited|30_0[TFilter,TFilterAsync](ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResultExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.ResultNext[TFilter,TFilterAsync](State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeResultFilters()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeNextResourceFilter>g__Awaited|25_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Rethrow(ResourceExecutedContextSealed context)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.InvokeFilterPipelineAsync()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Logged|17_1(ResourceInvoker invoker)
   at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
   at BSR.Beinni.Web.Public.BeinniWebPublicModule.<>c__DisplayClass13_0.<<OnApplicationInitialization>b__1>d.MoveNext() in C:\Dev\IBeinni\aspnet-core\src\BSR.Beinni.Web.Public\BeinniWebPublicModule.cs:line 314
--- End of stack trace from previous location ---
   at Volo.Abp.AspNetCore.Serilog.AbpSerilogMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Authorization.AuthorizationMiddleware.Invoke(HttpContext context)
   at Volo.Abp.AspNetCore.MultiTenancy.MultiTenancyMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Localization.RequestLocalizationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.RequestLocalization.AbpRequestLocalizationMiddleware.InvokeAsync(HttpContext context, RequestDelegate next)
   at Microsoft.AspNetCore.Builder.UseMiddlewareExtensions.<>c__DisplayClass6_1.<<UseMiddlewareInterface>b__1>d.MoveNext()
--- End of stack trace from previous location ---
   at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
   at Microsoft.AspNetCore.Diagnostics.ExceptionHandlerMiddleware.HandleException(HttpContext context, ExceptionDispatchInfo edi)
```

Could this be somehow connected to my yarn.lock files being with a 5.1.3 versions? How do you update that?

```
"@abp/aspnetcore.mvc.ui.theme.shared@~5.1.3":
  version "5.1.3"
  ...etc..
```

**Comment by sturlath** [comment]
I solved the problem at my end. I didn´t notice that yarn wasn´t installed so `abp install-libs` did create the libs folders and it looked OK but when I did make sure it was installed and was globally included there was much more that happened when I ran the install-libs command.

So do `npm install --global yarn` then `npm config get prefix` and copy that value into the path environment variable (press windows key and write env to get there).
